### ISSUE TITLE
The integration test framework should test the DEB and RPM installations of the agent

### ISSUE THREAD

**Issue by cmacknz** [elastic/elastic-agent#4086]
- Relates https://github.com/elastic/elastic-agent/issues/4084
- https://github.com/elastic/elastic-agent/issues/3784

At minimum we need to ensure that these packages install and can be enrolled in Fleet. The RPM and DEB tests should run in their own test group to ensure they run in parallel with other installation types.

The Fleet log ingestion test could be used for this purpose. https://github.com/elastic/elastic-agent/blob/main/testing/integration/logs_ingestion_test.go#L40

We should also test that they can be upgraded, see https://github.com/elastic/elastic-agent/issues/3784. This will require dedicated test as this is a new upgrade path, this part of the work could be split into a separate issue if needed.

**Comment by pierrehilbert** [comment]
Reopening as only the DEB part has been done for now
